source/documentation/deploying_services/use_a_custom_domain.md

```diff
@@ -15,79 +17,283 @@ You must configure the `cdn-route` service to use a subdomain. If you configure
-
-    ```bash
-    cf create-domain ORGNAME example.com
+    cf target -o ORG_NAME -s SPACE_NAME
     ```
 
-3. Map a subdomain route to your app:
+    where:
+    - `ORG_NAME` is the name of your org
+    - `SPACE_NAME` is the name of your space
+
+1. Create the required domain(s) in your organisation by running the following for each domain:
 
-    ```bash
-    cf map-route APPNAME example.com --hostname www
+    ```
+    cf create-domain ORG_NAME DOMAIN_NAME
     ```
 
-4. Create an instance of the `cdn-route` service by running the following command, replacing `my-cdn-route` with the name of your service instance, and `www.example.com` with your domain:
+1. Create an instance of the cdn-route service:
 
-    ```bash
-    cf create-service cdn-route cdn-route my-cdn-route \
-        -c '{"domain": "www.example.com"}'
+    ```
+    cf create-service cdn-route cdn-route SERVICE_INSTANCE
+    -c '{"domain": "SUBDOMAIN_LIST"}'
     ```
 
-    >This command includes `cdn-route` twice because `cdn-route` is the name of the service **and** the name of the service plan.
+    where:
+    - `SERVICE_INSTANCE` is the name you have chosen for your service instance
+    - `SUBDOMAIN_LIST` is a comma-separated list of your subdomain(s), for example `www.example.com,www.example.net` (do not include a comma after the final subdomain)
 
+    This command includes `cdn-route` twice because cdn-route is the name of the service and the name of the service plan.
 
-5. Get the DNS information required to configure your domain by running the following command, replacing `my-cdn-route` with the name of your service instance:
+1. Get the DNS information required to configure your subdomain:
 
-    ```bash
-    cf service my-cdn-route
     ```
-    This will output the DNS information, as per this example:
+    cf service SERVICE_INSTANCE
+    ```
+
+    This outputs the DNS information, as per this example:
 
     ```
     Last Operation
     Status: create in progress
-    Message: Provisioning in progress [www.example.com => origin-my-paas-app.london.cloudapps.digital];
-    CNAME or ALIAS domain www.example.com to d3nrs0916m1mk2.cloudfront.net or create TXT record(s):
-    name: _acme-challenge.www.example.com., value: ngd2suc9gwUnH3btm7N6hSU7sBbNp-qYtSPYyny325E, ttl: 120
+    Message: Provisioning in progress
+             [www.example.com,www.example.net => london.cloudapps.digital];
+             CNAME or ALIAS domain www.example.com,www.example.net to
+             d3nrs0916m1mk2.cloudfront.net or create TXT record(s):
+    name: _acme-challenge.www.example.com.,
+          value: ngd2suc9gwUnH3btm7N6hSU7sBbNp-qYtSPYyny325E, ttl: 120
+    name: _acme-challenge.www.example.net.,
+          value: 4W9jHJIhnfDnWBBAENph6Qr8aeoSV7FzDiEDL-s-o4Q, ttl: 120
+    ```
+
+1. Create the TXT records using the DNS information shown. These records:
+    - validate your subdomain
+    - issue the TLS certificate used to encrypt traffic
```

Comment:
Maybe this should say "allow us to issue the TLS...." - the DNS records don't actually issue the cert. I wonder whether the bullet list is necessary, we could simply say "These records validate your subdomain and allow us to issue the TLS certificate..."

Reply:
changed